builder/proxmox/common/step_finalize_config.go

```diff
@@ -17,18 +17,19 @@ import (
 // stepFinalizeTemplateConfig does any required modifications to the configuration _after_
```

Comment:
Minor nit, I'd suggest updating the documentation here just so the name of the step is correctly reflected.